heroku/cli
# refactor: migrate pg:wait to oclif/core

[Work Item](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001bDG1LYAW/view)

## Notes
This PR includes the removal of the pg-v5 package, since `pg:wait` was the last command left.

## Testing
- check out this branch locally and run `yarn` and `yarn build`
- Select an app and add a Heroku Postgres database. Select a Standard or Premium plan, since these take time to provision.
- Run `./bin/run pg:wait -a APP_NAME DATABASE_NAME`
    - It should wait until the database has been provisioned and then the command will end and you will see a message that the database is available.
